components/scream/scripts/test_all_scream.py

```diff
@@ -565,33 +565,40 @@ def generate_baselines(self, test, commit):
         print("Generating baseline for test {} with config '{}'".format(self._test_full_names[test], cmake_config))
         print("===============================================================================")
 
-        # We cannot just crash if we fail to generate baselines, since we would
-        # not get a dashboard report if we did that. Instead, just ensure there is
-        # no baseline file to compare against if there's a problem.
-        stat, _, err = run_cmd("{} {}".format(cmake_config, self._root_dir), from_dir=test_dir, verbose=True, dry_run=self._dry_run)
-        if stat!= 0:
-            print ("WARNING: Failed to configure baselines:\n{}".format(err))
-            return False
-
-        cmd = "make -j{} && make -j{} baseline".format(self._compile_res_count[test],self._testing_res_count[test])
-        if self._parallel:
-            start, end = self.get_taskset_id(test)
-            cmd = "taskset -c {}-{} sh -c '{}'".format(start,end,cmd)
+        success = True
 
-        stat, _, err = run_cmd(cmd, from_dir=test_dir, verbose=True, dry_run=self._dry_run)
+        try:
+            # We cannot just crash if we fail to generate baselines, since we would
+            # not get a dashboard report if we did that. Instead, just ensure there is
+            # no baseline file to compare against if there's a problem.
+            stat, _, err = run_cmd("{} {}".format(cmake_config, self._root_dir),
+                                   from_dir=test_dir, verbose=True, dry_run=self._dry_run)
+            if stat != 0:
+                print ("WARNING: Failed to configure baselines:\n{}".format(err))
+                success = False
```

Comment:
Shouldn't we immediately jump to the finally statement if we hit this case? There's no point in running `make` and `make baseline`, right?

Reply:
Yep. Good catch